### PR TITLE
Updates to some of the devicons options

### PR DESCRIPTION
**Pull request by cgfrost**
Minor updates following the refactoring and new support for devicons. Thanks for adding it, really appreciated.

### CODE REVIEW

File: lib/presets.ts [hunk 1]
```diff
@@ -348,39 +348,53 @@ export function setExtensions(set: RegisterIcon) {
...
 	});
 	set('dmg', {
 		[IconSet.ColorLight]: '#6f8696',
-		[IconSet.IconIon]: 'ion-briefcase'
+		[IconSet.IconIon]: 'ion-briefcase',
+		[IconSet.IconDev]: ['devicons devicons-netbeans', 17]
+	});
+	set('xpi', {
+		[IconSet.ColorLight]: '#5bac0d',
+		[IconSet.IconIon]: ['ion-briefcase', 16],
```
Comment (ivogabe) [inline lib/presets.ts]: 16 is the default size, so it's enough to only provide the class name.

File: lib/presets.ts [hunk 2]
```diff
@@ -398,7 +412,8 @@ export function setExtensions(set: RegisterIcon) {
 		'gruntrc',
 		'jsrc',
 		'pro',
-		'editorconfig'
+		'editorconfig',
+        'iml'
```
Comment (ivogabe) [inline lib/presets.ts]: Can you change spaces to tabs?

**Comment by ivogabe** [comment]
Glad you like it! Can you also add a Dockerfile to the examples directory?

**Comment by cgfrost** [comment]
I've made the requested changes. Thank you.

**Comment by cgfrost** [comment]
Awesome, thanks.

**Pull request merged**